DFC.App.JobGroups/Controllers/PagesController.cs

```diff
@@ -48,6 +50,14 @@ public class PagesController : Controller
             {
                 status = "PUBLISHED";
             }
+
+            if (this.configuration != null)
+            {
+                string expiryAppString = this.configuration.GetSection(ExpiryAppSettings).Get<string>();
+                if (double.TryParse(expiryAppString, out expiryInHours))
+                {
+                }
```

Comment:
While this is a bit better, from a defensive coding perspective, you are now running in to the same issue as if you had used `.Get<double>();`, because if the `expiryAppString` is not a number, you will set `expiryInHours` to 0.
I would therefore create a different variable for the parsed result, for example `expiryAppStringParseResult` (you are welcome to come up with a better name), and if the parsing is successful, you then set `expiryInHours` to the parsed value.